### DISCUSSION TITLE
Amplify CI: CI Build ID could not be generated and was not specified error

### DISCUSSION THREAD

**Discussion by anastasiyamazheika** [cypress-io/cypress#20337] [CI setup]
Hello, I have a question on how to parallelize tests in AWS Amplify. 

If I pass `--parallel` param, I get such an error:

```
We encountered an unexpected error talking to our servers.
There is likely something wrong with the request.
The --tag flag you passed was:
The --parallel flag you passed was: true
The server's response was:
StatusCodeError: 422
{
"message": "CI Build ID could not be generated and was not specified.",
"code": "INDETERMINATE_CI_BUILD_ID"
}
```

Thanks for help!

**Comment by anastasiyamazheika** [comment]
Hello, any updates on this?

**Comment by dgentle03** [comment]
Try adding '--ci-build-id $CODEBUILD_INITIATOR' to the build command.  

For example: 

npx cypress run --record --parallel --spec 'cypress/e2e/your-custom-folders/**/*-cy.js' --ci-build-id $CODEBUILD_INITIATOR

Cypress Documentation: https://docs.cypress.io/guides/continuous-integration/aws-codebuild
Real World Project: https://github.com/cypress-io/cypress-realworld-app/blob/develop/buildspec.yml
